icu_benchmarks/data/utils.py

```
@@ -0,0 +1,81 @@
+            data[Segment.outcome] = data[Segment.outcome].unique(subset=[group, sequence], keep=keep, maintain_order=True)
+        else:
+            data[Segment.outcome] = data[Segment.outcome].unique(subset=[group], keep=keep, maintain_order=True)
+        logging.warning(f"Removed {old_len - len(data[Segment.outcome])} duplicates from outcome data.")
+    return data, vars
+
+
+def modality_selection(
+    data: dict[pl.DataFrame], modality_mapping: dict[str], selected_modalities: list[str], vars
+) -> dict[pl.DataFrame]:
+    logging.info(f"Selected modalities: {selected_modalities}")
+    selected_columns = [modality_mapping[cols] for cols in selected_modalities if cols in modality_mapping.keys()]
```

Ruff (0.12.2)

Use `key in dict` instead of `key in dict.keys()`

Remove `.keys()`

(SIM118)